src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -221,6 +221,9 @@ TRANSFORMATIONS_API bool is_dequantization_subgraph(const Output<Node>& node);
 TRANSFORMATIONS_API bool can_eliminate_eltwise_node(const std::shared_ptr<Node>& eltwise,
                                                     const Output<Node>& constant,
                                                     const Output<Node>& non_constant_input);
+
+TRANSFORMATIONS_API bool is_constfoldable(const ov::Output<ov::Node>& output);
```

Comment:
@mateusztabaka I have one minor comment regarding the function name. As for me, this name is a bit misleading.
The function checks that output is on constant path, not constfoldable. In some cases `output` can be on constant path but not be constfoldable. E.g. if constant folding for some node on the path was disabled by the [other transformation](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/src/transformations/low_precision/mark_dequantization_subgraph.cpp#L79)
Maybe it's good idea to rename this function? What do you think?

Reply:
Right, I renamed the function to is_on_constant_path.